### PR TITLE
03 inlines and blocks

### PR DESCRIPTION
**Pull request by guillaume-boutin**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    background-color: rgba(0, 255, 0, 0.5);
+}
+nav {
+  background-color: pink;
+}
+/* select all the UL elements that are anywhere insde a NAV element */
+nav ul {
+    list-style: none;
+    /* not quite margin bottom, but anyway...*/
+    margin: 0 0 50px 0;
```
Comment (ziad-saab) [inline css/main.css]: @guillaume-boutin it's fine, no matter how you are setting the margin :) However, in this case why are you putting the margin on the `ul` element? Are you trying to space it out from something else?

**Comment by ziad-saab** [comment]
@guillaume-boutin one instruction was missed: adding some vertical spacing inside the `nav` so that elements are not stuck to its border. Please fix.

**Comment by guillaume-boutin** [comment]
Wrong element. My bad.

**Comment by ziad-saab** [comment]
@guillaume-boutin :+1:

**Pull request closed without merging**